### DISCUSSION TITLE
Best place  to find  up to date setup for PIA?

### DISCUSSION THREAD

**Discussion by JacobDK8200** [haugene/docker-transmission-openvpn#2290] [Q&A]
Hi

Just changed to PIA. Where do I find the best setup? With port forwarding, maybe wireguard?

Seems to be alot of outdated info out there.

Thanks!

**Comment by pkishino** [comment]
have you tried to search? the wiki describes how to setup, there is nothing special to it..for the wireguard beta, please search the discussion/issue forum
